book/01-introduction/sections/help.asc

```diff
@@ -1,46 +1,46 @@
-[[_git_help]]
-=== Getting Help
+[[_گرفتن_کمک]]
+=== کمک گرفتن
 
-If you ever need help while using Git, there are two equivalent ways to get the comprehensive manual page (manpage) help for any of the Git commands:
+اگر در هنگام استفاده از گیت همیشه به کمک نیاز دارید، برای دستیابی به راهنمای دستی جامع (صفحه من)هر یک از دستورات گیت دو روش معادل وجود دارد :
 
-[source,console]
+[منبع، کنسول]
```

Comment:
Don't translate the lines in `[...]`, they are asciidoctor formatting commands. To understand what needs to be translated and what needn't, please read how asciidoctor works here:

http://asciidoctor.org/docs/user-manual/